src/main/java/seedu/duke/userinterface/command/Done.java

```diff
@@ -1,8 +1,36 @@
 package seedu.duke.userinterface.command;
 
+import seedu.duke.exceptions.InvalidUserInputException;
+import seedu.duke.notebooks.NotebookShelf;
+import seedu.duke.tasks.Task;
+import seedu.duke.tasks.TaskList;
+import seedu.duke.userinterface.AppState;
+import seedu.duke.userinterface.CliMessages;
+import seedu.duke.userinterface.CliUserInterface;
+
 public class Done extends CliCommand {
+
+    public static final String COMMAND_WORD = "done";
+
+    private AppState appState;
+    private String arguement;
```

Comment:
argument* , same for the others

Reply:
Oh okay thanks! Will make edits! I think something went wrong during the merge conflict, was working fine on my end just now